docs/source/api/v4/multiple_servers_capabilities.rst

```diff
@@ -0,0 +1,187 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-v4-multiple_servers_capabilities:
+
+*********************************
+``multiple_servers_capabilities``
+*********************************
+
+.. versionadded:: 4.1
+
+``POST``
+========
+Inserts a list of :term:`Server Capability` associated to a server and vice versa i.e insert a list of :term:`Server` associated to a server capability.
```

Comment:
these terms should be plural; `Server Capabilities` and `Servers`, based on their usage